<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Fijaciones_tc-20240417115522" tests="1" file=".py" time="93.268" timestamp="2024-04-17T11:56:55" failures="0" errors="1" skipped="0">
	<testcase classname="Fijaciones_tc" name="test_grain_pipup" time="93.268" timestamp="2024-04-17T11:56:55" file="FijacionTdc.py" line="20">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=123.0.6312.123)
Stacktrace:
	GetHandleVerifier [0x00007FF642E17032+63090]
	(No symbol) [0x00007FF642D82C82]
	(No symbol) [0x00007FF642C1EC65]
	(No symbol) [0x00007FF642BFCA7C]
	(No symbol) [0x00007FF642C8D687]
	(No symbol) [0x00007FF642CA2AC1]
	(No symbol) [0x00007FF642C86D83]
	(No symbol) [0x00007FF642C583A8]
	(No symbol) [0x00007FF642C59441]
	GetHandleVerifier [0x00007FF6432125AD+4238317]
	GetHandleVerifier [0x00007FF64324F70D+4488525]
	GetHandleVerifier [0x00007FF6432479EF+4456495]
	GetHandleVerifier [0x00007FF642EF0576+953270]
	(No symbol) [0x00007FF642D8E54F]
	(No symbol) [0x00007FF642D89224]
	(No symbol) [0x00007FF642D8935B]
	(No symbol) [0x00007FF642D79B94]
	BaseThreadInitThunk [0x00007FFA9757257D+29]
	RtlUserThreadStart [0x00007FFA98AAAA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\FijacionTdc.py", line 96, in test_grain_pipup
    validate_text(self.driver,title_pinup_grain, title_pinup_grain_expected  )
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 454, in validate_text
    elemento = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 84, in _predicate
    return driver.find_element(*locator)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 742, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=123.0.6312.123)
Stacktrace:
	GetHandleVerifier [0x00007FF642E17032+63090]
	(No symbol) [0x00007FF642D82C82]
	(No symbol) [0x00007FF642C1EC65]
	(No symbol) [0x00007FF642BFCA7C]
	(No symbol) [0x00007FF642C8D687]
	(No symbol) [0x00007FF642CA2AC1]
	(No symbol) [0x00007FF642C86D83]
	(No symbol) [0x00007FF642C583A8]
	(No symbol) [0x00007FF642C59441]
	GetHandleVerifier [0x00007FF6432125AD+4238317]
	GetHandleVerifier [0x00007FF64324F70D+4488525]
	GetHandleVerifier [0x00007FF6432479EF+4456495]
	GetHandleVerifier [0x00007FF642EF0576+953270]
	(No symbol) [0x00007FF642D8E54F]
	(No symbol) [0x00007FF642D89224]
	(No symbol) [0x00007FF642D8935B]
	(No symbol) [0x00007FF642D79B94]
	BaseThreadInitThunk [0x00007FFA9757257D+29]
	RtlUserThreadStart [0x00007FFA98AAAA48+40]

]]></error>
		<system-out><![CDATA[El texto encontrado es  Fijaciones Habilitadas
Número de cuenta '484' ingresado y opción seleccionada con éxito!
Tiempo de espera agotado. El elemento no está presente o no es clickeable.
Tiempo de espera agotado. El elemento no está presente o no es visible.
]]></system-out>
	</testcase>
</testsuite>
